### DISCUSSION TITLE
Is there a way to open windows explorer on a terminal tab?

### DISCUSSION THREAD

**Discussion by gnllucena** [microsoft/terminal#15177] [Q&A] [answered]
I'm currently working on something that a I need a windows explorer tab (to delete, unzip folders) and a shell to run commands.

It would be tremendously convenient to have a windows explorer tab on my Terminal.

Is there a way to do this?

**Comment by j4james** [comment, marked as answer]
Not possible at the moment, but there has been discussion about potentially having panes in the terminal hosting other content (like a file explorer, or a web browser). See issue #997 for more info.
